### DISCUSSION TITLE
How to internationalise gui

### DISCUSSION THREAD

**Discussion by rizh42** [grafana/grafana#38537] [General]
Hello
My team wants me to translate grafana's gui into our language
Where I can find all GUI's text or is there any instruments to so this?

**Comment by torkelo** [comment]
We will be working on adding internationalization support in the coming months

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
